src/components/cards/TradeCardGP/TradeCardGP.vue

```diff
@@ -121,11 +123,12 @@ export default defineComponent({
   setup() {
     const highPiAccepted = ref(false);
     const priceImpact = ref(0);
-    const isSell = ref(true);
+    const exactIn = ref(true);
     const feeQuote = ref<FeeInformation | null>(null);
     const feeExceedsPrice = ref(false);
     const orderId = ref('');
     const orderMetadata = ref<OrderMetaData | null>(null);
+    const updatingQuotes = ref(false);
```

Comment:
This isn't used for now in the UI - but it will show that a quote is being updated (mostly for initial load/switching pairs)